link/uprobe.go

```diff
@@ -35,6 +35,13 @@ type Executable struct {
 	symbols map[string]elf.Symbol
 }
 
+// UprobeOptions defines additional parameters that will be used
+// when loading Uprobes.
+type UprobeOptions struct {
+	// Symbol offset. Must be provided in case of external symbols (shared libs).
+	Offset uint64
```

Comment:
This should define what happens in the case of valid "internal" symbol but offset given via options. Does the symbol offset take precedence or the value here?

Reply:
As of now, the provided offset takes precedence, I'll document it in the comment